tests/playwright-test/expect.spec.ts

```diff
@@ -126,19 +143,29 @@ test('should work with custom PlaywrightTest namespace', async ({ runTSC }) => {
   expect(result.exitCode).toBe(0);
 });
 
-test('should not expand huge arrays', async ({ runInlineTest }) => {
-  const result = await runInlineTest({
-    'expect-test.spec.ts': `
-      const { test } = pwt;
-      test('numeric ranges', () => {
-        const a1 = Array(100000).fill(1);
-        const a2 = Array(100000).fill(1);
-        a2[500] = 2;
-        test.expect(a1).toEqual(a2);
-      });
+test('should hide unrelevant conditional expect matchers', async ({ runTSC }) => {
+  test.fixme();
+  const result = await runTSC({
+    'a.spec.ts': `
+    const { test } = pwt;
+    test('custom matchers', async ({ page }) => {
+      await test.expect(page).toHaveURL('https://example.com');
+      await test.expect(page).toBe(true);
+      // @ts-expect-error
+      await test.expect(page).toBeEnabled();
+
+      await test.expect(page.locator('foo')).toBeEnabled();
+      await test.expect(page.locator('foo')).toBe(true);
```

Comment:
I'd think that this one is also an error.

Reply:
discussed offline: lets hide the default expect matchers for now. -> applied suggestion.